reactMovieApp/src/components/movieComponents/MovieImage.js

```diff
@@ -5,17 +5,16 @@ import PropTypes from 'prop-types';
 
 const IMG_URL = CONFIG.IMG_URL;
 
-function MovieImage(movie) {
-
+function MovieImage({image}) {
```

Comment:
how come? why movie description is arrow func, but here you use functions expression